ssg/constants.py

```diff
@@ -193,8 +193,8 @@
 }
 
 FULL_NAME_TO_PRODUCT_MAPPING = {
-    "Alinux 2": "alinux2",
-    "Alinux 3": "alinux3",
```

Comment:
There are multiple occurences of this name is  `CMakeLists.txt` and `constants.xslt`, please fix them as well.

Reply:
> constants.xslt

Hi @jan-cerny ，

Thanks for your apply. However
- In [Alinux2/constants.xslt](https://github.com/ComplianceAsCode/content/blob/master/products/alinux2/transforms/constants.xslt#L5) and [Alinux3/constants.xslt](https://github.com/ComplianceAsCode/content/blob/master/products/alinux3/transforms/constants.xslt#L5) I already fixed them.
- In [CMakeLists.txt](https://github.com/ComplianceAsCode/content/blob/master/CMakeLists.txt) , it doesn't include the full name of the product.

Thanks again. If you have any questions, please let me know.

Reply:
These might be the place that @jan-cerny is talking about in [CMakeLists.txt](https://github.com/ComplianceAsCode/content/blob/master/CMakeLists.txt)

ALinux2
* https://github.com/ComplianceAsCode/content/blob/master/CMakeLists.txt#L69
* https://github.com/ComplianceAsCode/content/blob/master/CMakeLists.txt#L263

ALinux3
* https://github.com/ComplianceAsCode/content/blob/master/CMakeLists.txt#L70
* https://github.com/ComplianceAsCode/content/blob/master/CMakeLists.txt#L264

It appears we are inconstant with short name vs. long name in this file. @jan-cerny thoughts?

Reply:
Hi @Mab879 , 

Thanks for your reply. Now I fix all inconstant places. @jan-cerny